### PR TITLE
Add ÖBB Railjet support

### PR DESCRIPTION
**Pull request by raccube**
Tried to keep the names similar to the DB one

Kräuter Gewürze und Kräuter Nüsse :E

### CODE REVIEW

File: ambient.d/trains/AT_OEBB.fish [hunk 1]
```diff
@@ -0,0 +1,28 @@
+ambient_is_ssid "OEBB"; or exit
+
+curl -s https://railnet.oebb.at/assets/modules/fis/combined.json | jq -r '"
+AMBIENT_AT_OEBB_SPEED=\(.operationalMessagesInfo.speed)
+AMBIENT_AT_OEBB_SITUATION=\(.operationalMessagesInfo.situation)
+AMBIENT_AT_OEBB_DELAY=\(.operationalMessagesInfo.delay)
+AMBIENT_AT_OEBB_SPEED_UNIT=km/h
```
Comment (liclac) [inline ambient.d/trains/AT_OEBB.fish:7]: Minor thing - could we move this just below `SPEED`?

File: ambient.d/trains/AT_OEBB.fish [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+AMBIENT_AT_OEBB_LATITUDE=\(.mapInfo.latitude)
+AMBIENT_AT_OEBB_LONGITUDE=\(.mapInfo.longitude)
+AMBIENT_AT_OEBB_ORIENTATION=\(.mapInfo.orientation)
+AMBIENT_AT_OEBB_SERVER_TIME=\(.operationalMessagesInfo.time | strptime("%s") | todate)
+AMBIENT_AT_OEBB_TRAIN_TYPE=\(.currentJourney.trainType)
+AMBIENT_AT_OEBB_VZN=\(.currentJourney.tripNumber)
```
Comment (liclac) [inline ambient.d/trains/AT_OEBB.fish]: I think this should actually be called `TRIP_NUMBER`, I don't actually know what `vzn` means, but I assume it's some sort of German abbreviation that DB like.

**Review by liclac — CHANGES_REQUESTED** [review]
This is looking great!

File: ambient.d/trains/AT_OEBB.fish [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+AMBIENT_AT_OEBB_STOP_FIRST=\(.currentJourney.start.all)
+AMBIENT_AT_OEBB_STOP_FIRST_DEPART_SCHEDULED=\((.forecastTimes | first).departureSchedule)
+AMBIENT_AT_OEBB_STOP_LAST=\(.currentJourney.destination.all)
+AMBIENT_AT_OEBB_STOP_LAST_ARRIVE_SCHEDULED=\((.forecastTimes | last).arrivalSchedule)
+AMBIENT_AT_OEBB_STOP_LAST_ARRIVE_ACTUAL=\((.forecastTimes | last).arrivalForecast)
+"'
+
+curl -s https://railnet.oebb.at/assets/modules/fis/combined.json | jq -r '"
+AMBIENT_AT_OEBB_STOP_NEXT=\({stationList: .stationList[], station_id: .operationalMessagesInfo.situationStation} | select(.stationList.id == .station_id).stationList.name.all)
+AMBIENT_AT_OEBB_STOP_NEXT_DEPART_SCHEDULED=\({forecastTimes: .forecastTimes[], station_id: .operationalMessagesInfo.situationStation} | select(.forecastTimes.station_id == .station_id) | .forecastTimes.departureSchedule)
```
Comment (liclac) [inline ambient.d/trains/AT_OEBB.fish:22]: Nice hack 😄

**Comment by raccube** [comment]
Here are the responses I had saved locally:
https://gist.github.com/mltnhm/5b7cc6b2e8fe268e5155d4f6d0d81aa5

File: ambient.d/trains/AT_OEBB.fish (hunk 2 above)
Comment (raccube) [reply ambient.d/trains/AT_OEBB.fish]: There was also a field called lineNumber which seemed to have the same contents, so I've added that in as well

**Comment by liclac** [comment]
:raised_hands:

**Pull request merged**